spec/models/user_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  subject { FactoryBot.create :user }
+
+  describe "associations" do
+    it { should have_many(:orders).dependent(:destroy) }
+  end
+
+  context "validations" do
```

Comment:
context, describle và it khác gì nhau e

Reply:
- Describe dùng để nhóm các test case theo class, module, method hoặc theo action của controller
- Context dùng để nhóm các điều kiện trong Example
- it dùng để định nghĩa 1 test case